### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by BipBobBoop**
[DEMO LINK](https://BipBobBoop.github.io/react_goods-selector-js/)

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  'Apple',
+  'Bread',
+  'Fish',
+  'Honey',
+  'Jam',
+  'Garlic',
+];
+
+export const App = () => {
+  const [selectedGood, setValue] = useState('Jam');
```
Comment (Anastasiia-Svintsova) [inline src/App.jsx:19]: 1) You've already had the 'Jam' in the goods array.
2)  It's recommended to name the setter function based on the specific state variable it's associated with
```suggestion
  const [selectedGood, setSelectedGood] = useState(goods[8]);
```

File: src/App.jsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+  return (
+    <main className="section container">
+      {selectedGood
+        ? (
```
Comment (Anastasiia-Svintsova) [inline src/App.jsx:24]:
```suggestion
      {selectedGood ? (
```

File: src/App.jsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+          <h1 className="title is-flex is-align-items-center">
+            {`${selectedGood} is selected`}
+            <button
+              data-cy="ClearButton"
+              type="button"
+              className="delete ml-3"
+              onClick={() => setValue('')}
```
Comment (Anastasiia-Svintsova) [inline src/App.jsx:31]: You use this logic for the remove button as well. So, move this logic to a separate function to not repeat yourself

File: src/App.jsx [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+            />
+          </h1>
+        )
+        : (
+          <h1 className="title is-flex is-align-items-center">
+            No goods selected
+          </h1>
+        )
+    }
```
Comment (Anastasiia-Svintsova) [inline src/App.jsx:40]:
```suggestion
        ) : (
          <h1 className="title is-flex is-align-items-center">
            No goods selected
          </h1>
        )}
```

File: src/App.jsx [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+            return (
+              <tr
+                data-cy="Good"
+                className={isSelectedGood && 'has-background-success-light'}
+              >
+                <td>
+                  <button
+                    data-cy={isSelectedGood ? 'RemoveButton' : 'AddButton'}
+                    type="button"
+                    className={isSelectedGood ? 'button is-info' : 'button'}
```
Comment (Anastasiia-Svintsova) [inline src/App.jsx:55]: In my opinion, it's better to create 2 separate buttons to separate the logic. Let's imagine that your app will grow and you need to use these buttons separately. 
But both approaches can be valid, so it's up to you

**Review by Anastasiia-Svintsova — CHANGES_REQUESTED** [review]
Good start! Let's add a few fixes

File: src/App.jsx (hunk 5 above)
Comment (BipBobBoop) [reply src/App.jsx]: не можу нажаль закинути пул реквест в потрібну директорію... її немає в списку:
image

**Pull request closed without merging**